README.md

```diff
@@ -48,10 +50,19 @@ Install from the [Microsoft Store's PowerToys page][microsoft-store-link]. You m
 ### Via WinGet
 Download PowerToys from [WinGet][winget-link]. To install PowerToys, run the following command from the command line / PowerShell:
 
+#### User scope installer [default]
 ```powershell
 winget install Microsoft.PowerToys -s winget
 ```
 
+#### Machine-wide scope installer
+
+```powershell
+winget install --scope machine Microsoft.PowerToys -s winget
+```
+
+Updating PowerToys via winget will respect current PowerToys installation scope.
```

Comment:
Move this to "via winget" section

Reply:
wdym? It's at the bottom of via winget section ? To move it to top into the first paragraph?